src/js/service/PolygonService.js

```diff
@@ -0,0 +1,74 @@
+import data from '../constants/figures.json';
+import { connectingPoints } from '../math/compound';
+import { getCoordinates } from '../draw/getCoordinates';
+import Polygon from '../models/Polygon';
+
+export default class PolygonService {
+    initializeFigures() {
+        const figures = [];
+        const forms = data.data;
+        let indent = 0;
+
+        forms.forEach((form, i) => {
+            const dots = initializeDots();
+            const numberOfCorner = initializeCorners(form);
+            const rightWays = connectingPoints(dots, numberOfCorner);
+            const figure = getCoordinates(rightWays, dots, indent);
+
+            figures.push(new Polygon(i, figure));
+            indent += 350;
```

Comment:
don't use 'magic numbers' please. Add `const INDENT = 350` above the class declaration, please